tests/specs/session-replay/rrweb-configuration.e2e.js

```diff
@@ -236,6 +236,19 @@ describe.withBrowsersMatching(notIE)('RRWeb Configuration', () => {
 
       expect(JSON.stringify(body).includes('testing')).toBeFalsy()
     })
+
+    it('block_selector: should not extend on empty string argument', async () => {
```

Comment:
This is IMHO something that we should move to a unit test.

Reply:
converted to a jest test